components/layouts/blog/blog-homepage-section.hbs

```diff
@@ -163,6 +163,9 @@
             </div><!-- .content-slider__cards-list .grid-col-content__four--homepage-blog -->
 
         </div><!-- .content-slider-blog-->
+        <div class="content-block--homepage-blog--blog-button">
```

Comment:
This has two modifiers in the class name.

Reply:
Fixed